src/pendulum_constant_torque_demo.py

```diff
@@ -27,6 +27,11 @@
 visualizer = builder.AddSystem(PendulumVisualizer())
 builder.Connect(pendulum.get_output_port(0), visualizer.get_input_port(0))
 
+signalLogRate = 60
+signalLogger = builder.AddSystem(SignalLogger(2))
+signalLogger._DeclarePeriodicPublish(1. / signalLogRate, 0.0)
```

Comment:
You orobably also need to turn off the publish on every timestep (as i did in the constant torque demo)?

Reply:
Not sure what you mean -- this comment is in the constant torque demo, which does have every-time-step publishing disabled?

Reply:
My bad